### DISCUSSION TITLE
Failed to write, and not due to blocking: Broken pipe in chunk cache

### DISCUSSION THREAD

**Discussion by kadhamecha-conga** [grafana/mimir#6362] [General]
hi team,

why i'm getting `Failed to write, and not due to blocking: Broken pipe ` this error ?
what's probable cause and how to fix this ?

thanks

**Comment by a5r0n** [comment]
Same here, any idea what could be wrong?

**Comment by kavirakesh14** [comment]
How to resolve this?

**Comment by pstibrany** [comment]
Do you have any more details about this error? Full log message would be helpful, component which logged this could also help us understand what's going on. I haven't seen this error before.
